app/src/main/res/values-pt-rBR

```diff
@@ -0,0 +1,401 @@
+    <string name="internal_firmware_settings">Configurações internas de firmware</string>
+    <string name="internal_firmware_settings_summary">Ajustar as configurações do firmware interno.</string>
+    <string name="custom_bios_firmware">BIOS e firmware personalizados</string>
+    <string name="custom_bios_firmware_description">O uso de um BIOS e firmware personalizados permite que você inicie e navegue no firmware do DS sem carregar uma ROM, assim como em um sistema real</string>
+    <string name="bios_directory">Diretório BIOS DS</string>
+    <string name="dsi_bios_directory">Diretório BIOS DSi</string>
+    <string name="show_boot_screen">Mostrar tela de inicialização</string>
+    <string name="mac_address">Endereço MAC</string>
+    <string name="randomize_mac_address">Randomizar endereço MAC</string>
+    <string name="randomize_mac_address_summary">Quando ativado, um endereço MAC aleatório é usado a cada inicialização. Isso pode fazer com que alguns jogos pensem que estão sendo executados em um sistema diferente e podem interromper eventos baseados em tempo.</string>
+    <string name="generate_new_mac_address">Gerar novo</string>
+    <string name="generate_new_mac_address_description">Gerando um novo endereço MAC fará com que os jogos pensem que estão rodando em um sistema diferente.</string>
+    <string name="enable_jit">Ativar JIT</string>
+    <string name="enable_jit_summary">Melhora o desempenho, mas pode causar congelamentos ou travamentos.</string>
+    <string name="jit_not_supported">O JIT é suportado apenas em dispositivos de 64 bits.</string>
+    <string name="threaded_rendering">Renderização encadeada</string>
+    <string name="fps_counter_position">Posição do contador FPS</string>
+    <string name="threaded_rendering_summary">Melhora o desempenho em jogos 3D quando ativado, mas pode causar falhas gráficas.</string>
+    <string name="category_video">Vídeo</string>
+    <string name="category_video_summary">Filtro, renderização encadeada, contador de FPS</string>
+    <string name="filter">Filtro</string>
+    <string name="category_audio">Áudio</string>
+    <string name="category_audio_summary">Volume, latência, fonte do microfone</string>
+    <string name="enable_sound">Ativar som</string>
+    <string name="interpolation">Interpolação</string>
+    <string name="bitrate">Taxa de bits</string>
+    <string name="volume">Volume</string>
+    <string name="volume_percentage">%1$d%%</string>
+    <string name="audio_latency">Latência do Áudio</string>
+    <string name="microphone_source">Fonte do microfone</string>
+    <string name="controller_layout">Layout do Controle</string>
+    <string name="category_save_files">Salvar Arquivos</string>
+    <string name="category_save_files_summary">Salvar arquivo &amp; salvar locais do estado</string>
+    <string name="save_next_rom">Salvar arquivo próximo à ROM</string>
+    <string name="save_file_directory">Salvar diretório de arquivo</string>
```

Comment:
This string is not supposed to be used as a verb. Instead, it indicates the directory in which save files will be stored. As such, I believe that a better translation would be "Diretório de arquivos de salvamento".

I don't really like the name "arquivos de salvamento", but I don't have a better idea 😔

Reply:
> This string is not supposed to be used as a verb. Instead, it indicates the directory in which save files will be stored. As such, I believe that a better translation would be "Diretório de arquivos de salvamento".
> 
> I don't really like the name "arquivos de salvamento", but I don't have a better idea pensive

I don't think it would change much if it were to " Diretório de salvamento de arquivos" 😐